04/app.js

```diff
@@ -2,4 +2,71 @@ document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     console.log('DOM');
-}
\ No newline at end of file
+    const form = document.querySelector('.form');
+
+    form.addEventListener('submit', showWeather)
+}
+
+function showWeather(e) {
+    e.preventDefault();
+    const formValues = getValueForm();
+    const {latitude, longitude} = formValues;
+    showCordinate(latitude, longitude);
+    const data = dataWeather(latitude, longitude);
+
+    data.then(data => {
+        console.log(data);
+        const {lon, lat, temp, weather: {description}} = data.data[0];
+        console.log(temp, description);
+        return {lon, lat, description, temp}
+    })
+    .then((res) => {
+        const weatherSummery = document.querySelector('.weather__summary');
+        const currentDestTemp = document.querySelector('.weather__temperature');
+
+        weatherSummery.innerText = res.description;
+        currentDestTemp.innerText = res.temp;
+    })
```

Comment:
Tutaj również mógłbyś wstawić funkcje nazwaną, a nie tak jak teraz jest anonimowa.
Czyli chodzi mi o coś takiego `.then(myFunctionName)`, i potem `const myFunctionName = res => { /* ... */}`;
To tylko tak informacyjnie piszę, aba rozwiązania są ok.